src/components/MainContent/index.js

```diff
@@ -221,7 +229,7 @@ export const MainContent = ({ styles }) => {
                 onClick={() => {
                   document.getElementsByClassName(
                     'inner-div'
-                  )[0].scrollLeft -= 100
+                  )[0].scrollLeft -= 360
```

Comment:
@bhavya2611 I am pretty confident about all of the other changes in this PR - but wanted to get your approval on this change in particular, please. =)

This is about smoothing out the behavior of the left arrow in the SDK and tools module on the home page. For example, on the live site https://xmtp.org/, use the right arrow to scroll to the right. Then click the left arrow to scroll to the left - it's a smaller scroll increment than when scrolling right.

See the effect of this change using this preview URL: https://junk-range-possible-git-add-qs-app-xmtp-labs.vercel.app/. 

Seems to work, but wanted to be sure this is an okay code change! TY

Reply:
Looks good, it might not be the best idea to have a constant number here, but I think it is working fine so we are good